cloudify_types/cloudify_types/component/operations.py

```diff
@@ -333,3 +333,19 @@ def refresh(**kwargs):
                      deployment.get('id') or
                      ctx.instance.id)
     populate_runtime_with_wf_results(client, deployment_id)
+
+
+@operation(resumable=True)
+@errors_nonrecoverable
+def check_drift(**kwargs):
+    # We might need an endpoint to get the drift and the status summary for
+    # the entire deployment so we don't need to review all node instances
+    # system-properties
+    pass
+
+
+@operation(resumable=True)
+@errors_nonrecoverable
+def check_status(**kwargs):
+    # run the check status workflow and collect the status
```

Comment:
it's I think already going to be automatically run when we do run check_status on _this_ deployment, because check_status is cascading